test/acceptance/features/steps/steps.py

```diff
@@ -190,6 +206,78 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
 
 
 # STEP
+is_serverless_operator_running_step = u'Openshift Serverless Operator is running'
+
+
+@given(is_serverless_operator_running_step)
+def given_serverless_operator_is_running(context):
+    """
+    Checks if the serverless operator is up and running
+    """
+    serverless_operator = ServerlessOperator()
+    if not serverless_operator.is_running():
+        print("Serverless operator is not installed, installing...")
+        serverless_operator.install_operator_subscription() | should.be_truthy.desc(
+            "serverless operator subscription installed")
+        serverless_operator.is_running(wait=True) | should.be_truthy.desc("serverless operator installed")
+    print("Serverless operator is running!!!")
+    context.serverless_operator = serverless_operator
+
+
+# STEP
+is_quarkus_s2i_image_present_step = u'Quarkus s2i builder image is present'
```

Comment:
see the comment above.

Reply:
We could apply the same for other steps as well. As this var that holds the description of the step can be in given or any other keywords like then